src/front/wgsl/mod.rs

```diff
@@ -439,6 +453,15 @@ impl<'a> Error<'a> {
                 labels: vec![(span.clone(), "expression is not a reference".into())],
                 notes: vec![],
             },
+            Error::InvalidAssignment{ ref span, ty} => ParseError {
+                message: "invalid left-hand side of assignment".into(),
+                labels: vec![(span.clone(), "cannot assign to this expression".into())],
+                notes: match ty {
+                    InvalidAssignmentType::Swizzle => vec!["WGSL does not support assignments to swizzles".into(), "consider assigning each component individually".into()],
```

Comment:
I'm surprised that `cargo fmt` doesn't wrap those long lines. The CI should have caught that, though, so maybe I'm missing something.